tests/integration/pilot/vm/vm_test.go

```diff
@@ -81,8 +81,6 @@ spec:
 				{
 					Name:     "http",
 					Protocol: protocol.HTTP,
-					// Due to a bug in WorkloadEntry, service port must equal target port for now
```

Comment:
I think if unset, we default instance == service port? based on https://github.com/istio/istio/blob/365a94720d5a5290fafeaee85d49ad83f144be1e/pkg/test/framework/components/echo/common/config.go#L96. So we may need to explicitly make them different.

And also change this usage: https://github.com/istio/istio/blob/37119973c952151e269110170f2fda8c6a34fb5e/tests/integration/security/util/reachability/context.go#L101